main.go

```diff
@@ -11,9 +12,15 @@ var version = flag.Bool("version", false, "print version information and exit")
 var list = flag.Bool("list", false, "list mode")
 var host = flag.String("host", "", "host mode")
 
+type Config struct {
+	TfState string `env:"key=TI_TFSTATE"`
```

Comment:
The key is just whatever environment variable you want it to be.  In case at some point TFSTATE is used by Terraform, I tried to ensure uniqueness to this app by using "TI_" as a prefix (terraform-inventory).